docs/getting-started/compress.md

```diff
@@ -21,17 +21,11 @@ For this guide, we'll use the `TinyLlama` model and the `open_platypus` dataset
 
 ## Select a Quantization Method and Scheme
 
-LLM Compressor supports several quantization methods and schemes, each with its own strengths and weaknesses. The choice of method and scheme will depend on your specific use case, hardware capabilities, and desired trade-offs between model size, speed, and accuracy.
+LLM Compressor supports several quantization methods and schemes, each with its own strengths and weaknesses. The choice of method and scheme will depend on your specific use case, hardware capabilities, and chosen trade-offs between model size, speed, and accuracy.
 
-Some common quantization schemes include:
+Supported quantization schemes include PTQ, W4A16, W8A8‑INT8, W8A8‑FP8, and Sparsification. For a more detailed overview of available quantization schemes, see [Compression Schemes](../guides/compression_schemes.md).
```

Comment:
sparsification is a compression scheme separate from quantization, the two can be done in tandem. PTQ is post-training-quantization, little bit different:
```suggestion
Supported compression schemes include quantization into W4A16, W8A8‑INT8, and W8A8‑FP8 formats, and sparsification. For a more detailed overview of available quantization schemes, see [Compression Schemes](../guides/compression_schemes.md).
```